app/agents/voice/automatic/prompts/system/personalization.py

```
@@ -11,4 +15,110 @@ def append_user_info(user_name: str) -> str:
+
+            if user_context_data.get("error"):
+                return append_user_info(user_name)
+
+            user_context = user_context_data.get("context", "")
+
+            if user_context:
+                return append_user_info(user_name, user_context)
+            else:
+                return append_user_info(user_name)
+
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)